### DISCUSSION TITLE
1.63 Hotfix update?

### DISCUSSION THREAD

**Discussion by Mikeyjr341** [wopss/RED4ext#25] [Q&A] [answered]
Any idea when the red4EXT update will be ready for the hotfix? I attempted to build what you had committed after the hotfix but I end up not have dependencies fulfilled and VS not cooperating.

**Comment by vortikon** [comment]
> Any idea when the red4EXT update will be ready for the hotfix? I attempted to build what you had committed after the hotfix but I end up not have dependencies fulfilled and VS not cooperating.

Go to the actions section on this git and download the file red4ext_debug_57858c6 that's the new compile/build for the hotfix.

**Comment by wopss** [comment, marked as answer]
This should be fixed in [v1.15.0](https://github.com/WopsS/RED4ext/releases/tag/v1.15.0).

I released it just now.
